typescript/src/folder/ui-folder-info/FolderInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./FolderInfo.module.scss";
+import classNames from "classnames/bind";
+
+const cx = classNames.bind(styles);
+
+interface props {
```

Comment:
타입은 파스칼 케이스로 작성하는게 컨벤션입니다~
```suggestion
interface Props {
```